Cargo.toml

```diff
@@ -7,15 +7,16 @@ license = "Apache-2.0"
 repository = "https://github.com/oxidecomputer/diesel-dtrace.git"
 
 [dependencies]
-diesel = { version = "2.1.0", features = [ "r2d2", "i-implement-a-third-party-backend-and-opt-into-breaking-changes" ] }
+diesel = { version = "=2.1.6", features = [ "r2d2", "i-implement-a-third-party-backend-and-opt-into-breaking-changes" ] }
 serde = "1"
 usdt = "0.5"
 uuid = { version = ">=0.8.0, <2.0.0", features = [ "v4", "serde" ] }
 
 [dev-dependencies]
-async-bb8-diesel = { git = "https://github.com/oxidecomputer/async-bb8-diesel" }
+## match omicron
```

Comment:
why is omicron pinned to a rev here? how do we envision advancing everyone forward?

Reply:
@ahl I'm not sure why omicron is pinned to a rev here. I'm going off what's in `main`, which hits at this: https://github.com/oxidecomputer/omicron/commit/b3d641a2dd3bb3f3dd68a413ddd670e3dbcc0b5c.

Reply:
actually, the culprit may be: https://github.com/oxidecomputer/async-bb8-diesel/blob/master/Cargo.toml#L26 which has diesel 2.2.